test/Limiters/limiter.jl

```diff
@@ -194,8 +211,9 @@ end
         Limiters.compute_bounds!(limiter, ρq_ref, ρ)
         Limiters.apply_limiter!(ρq, ρ, limiter)
 
-        @test parent(ρq)[:, :, 1, 1] ≈
-              [FT(0.0) FT(0.0); FT(0.950005) FT(0.950005)] rtol = 10eps(FT)
+        @test Array(parent(ρq))[:, :, 1, 1] ≈
```

Comment:
Why is an explicit `Array` needed here?

Reply:
It's needed (if data resides on the gpu) to move the data from the CPU to the GPU

Reply:
Got it, thanks!